### PR TITLE
Update README for 9.3.3 release

### PR DESCRIPTION
**Pull request by bourtemb**
Update documentation links
Update requirements
Added information on how to generate the documentation
Improve migration database procedures
Fix typos
Remove obsolete Lyx part

### CODE REVIEW

File: assets/README [hunk 1]
```diff
@@ -351,6 +361,19 @@ make all
 
 make install
 
+If you get some compilation errors like:
+error: cannot convert ‘zmq::socket_t’ to ‘void*’ in assignment
+this is due to an incompatible zmq.hpp file (probably provided with your Linux distribution).
+As a work-around, you will need to install cppzmq via:
```
Comment (mliszcz) [inline assets/README:367]: Is this still necessary after tango-controls/cppTango#561?

**Review by mliszcz — COMMENTED** [review]
Looks good, I have just one question.

File: assets/README (hunk 1 above)
Comment (t-b) [reply assets/README:367]: @mliszcz @bourtemb No!

File: assets/README [hunk 2]
```diff
@@ -504,6 +537,15 @@ If Tango is properly installed at your site, a copy of the documentation should
...
 
+Sphinx is used to generate the tango documentation html pages.
+Nevertheless, if you want to re-generate the documentation, you can use 
+the command "make html" in doc/src directory.
+This requires Sphinx and IPython to be installed on your host.
+On a Debian-like system, you can install these tools using the following 
+commands:
+sudo apt install -y python python-pip
+pip install Sphinx ipython
+
```
Comment (t-b) [inline assets/README:548]: `Sphinx -> sphinx` in the pip command. Is ipython really a requirement? I might be convenient to use but I don't think it is strictly required.

Comment (mliszcz) [reply assets/README:548]: As for the IPython, it is necessary to build the docs:

https://github.com/tango-controls/tango-doc/blob/982785add4e1e1d497106bdc39f5ced65990d57c/requirements.txt#L3

https://github.com/tango-controls/tango-doc/blob/982785add4e1e1d497106bdc39f5ced65990d57c/source/conf.py#L19

File: assets/README (hunk 1 above)
Comment (bourtemb) [reply assets/README:367]: Good point but I think it is still necessary for this release since we provide cppTango 9.3.3 with it.

**Review by mliszcz — APPROVED** [review]

File: assets/README (hunk 2 above)
Comment (t-b) [reply assets/README:548]: I've fixed the `sphinx` casing.

**Review by t-b — APPROVED** [review]

**Pull request merged**